bin/oneoff/populate_discount_codes.rb

```diff
@@ -110,3 +110,31 @@ def create_discount_eligible_section(teacher, section_name = 'Discount Eligible
   end
   section
 end
+
+# This will generate a report on the status of all of our discount applications.
+# It will print out a bunch of rows, one per line. You can then copy paste this
+# into google sheets. After pasting, click on the little icon that shows up and
+# select "Split text into columns"
+def discount_code_reporting(filepath)
+  rows = []
+  rows << [
+    'User Id', 'User Name', 'Unit6Intention', 'AdminOverrode',
+    'User signature', 'Sign Date', 'Full Discount', 'Discount Code', 'Last updated',
+    'School Id'
+  ].join(',')
+  CircuitPlaygroundDiscountApplication.all.each do |app|
+    rows << [
+      app.user_id,
+      app.user.name,
+      app.unit_6_intention,
+      app.admin_set_status,
+      app.signature,
+      app.signed_at,
+      app.full_discount,
+      app.circuit_playground_discount_code.try(:code),
+      app.updated_at,
+      app.school_id,
+    ].join(',')
+  end
+  puts rows
```

Comment:
Just experimenting: I think you can do this using the CSV module's [shortcut interface](http://ruby-doc.org/stdlib-2.0.0/libdoc/csv/rdoc/CSV.html#class-CSV-label-Shortcut+Interface), but I'm not sure it's actually any better:

```rb
def discount_code_reporting
  CSV do |csv|
    csv << [
      # Column names
    ]
    CircuitPlaygroundDiscountApplication.all.each do |app|
      csv << [
        # columns for row
      ]
    end
  end
end
```